alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java

```diff
@@ -58,9 +63,8 @@ public String createVersionableNode(String name) {
     @Override
     public void deleteNode(String liveNodeUuid) {
```

Comment:
AFAIK we don't have the functionality yet to implement this "in a clean way".

Feature request: `deleteNode` in Ditto.

Reply:
ACK

Reply:
can you make an issue in ditto referencing this PR?

Reply:
If you would like that :-)

Reply:
Toon acking is fine as well :)